### ISSUE TITLE
Does owner automatically get rewards and payments? 

### ISSUE THREAD

**Issue by steven004** [filecoin-project/specs#1184]
In https://spec.filecoin.io/#systems__filecoin_nodes__repository__key_store : 

`The owner keypair is provided by the miner ahead of registration and its public key associated with the miner address. Block rewards and other payments are made to the ownerAddress.`

In lotus implementation, I think block rewards are kept as part of miner's balance, unless withdraw by the owner. Is that right?

**Comment by Stebalien** [comment]
Correct. Payments are made to the miner, but only the ownerAddress can withdraw them.

**Comment by hugomrdias** [comment]
@yiannisbot should we review this content to make it clear ?
